virtio-devices/src/block.rs

```diff
@@ -522,6 +528,51 @@ impl Block {
         );
         self.writeback.store(writeback, Ordering::Release);
     }
+
+    #[cfg(fuzzing)]
+    pub fn wait_for_all_events_once(&self) -> std::result::Result<(), EpollHelperError> {
```

Comment:
I'm not a huge fan of this approach (instrumenting directly inside the block implementation), but it works well for what you're trying to achieve so let's see what others think about it.
/cc @rbradford @liuw

Reply:
Will it be possible to make it a normal function that takes a `&Block`?

Reply:
Seems a bit intrusive to me. The interesting bits are `BlockEpollHandler::handle_event` and the functions it calls. Can it or they be fuzzed instead?

Reply:
Unfortunately, the attempt with `block.wait_for_all_events_once()` did not work, as there is always a window that the `queue_evt` has been delivered/read while the `complete_evt` is not signaled. So that commit was dropped.

Reply:
> The interesting bits are `BlockEpollHandler::handle_event` and the functions it calls. Can it or they be fuzzed instead?

I think that's doable. That is essentially writing a stub for the `block.activate()` function but reusing the `BlockEpollHandler` (which is the real fuzzing target). Likely I will need to expose some more public functions/structs so that they are accessible outside the virtio-devices crate. 

I would prefer to try this in a separate PR and land this one first. In this way, oss-fuzz will start to perform meaningful fuzzing on the virtio-block. WDYT?

Reply:
Okay. I guess this new function is only exposed when using fuzzing but it does add a lot of complexity to the code.